pkg/apis/direct.csi.min.io/v1alpha1/types.go

```diff
@@ -116,11 +117,15 @@ type DirectCSIVolume struct {
 	metav1.ObjectMeta `json:"metadata"`
 
 	// +optional
-	OwnerDrive *DirectCSIDrive `json:"ownerDrive,omitempty"`
+	OwnerDrive string `json:"ownerDrive,omitempty"`
```

Comment:
We should store a reference to the object (its name), rather than a copy of it. Otherwise we'll have multiple sources of truth